### PR TITLE
Zelbinon namer

### PR DESCRIPTION
**Pull request by Vasily2013**
## About The Pull Request
This PR renames the torpedo casing in the torp components file from 30mm to 70cm. 

## Why It's Good For The Game

a 30mm torpedo is stupid and undersized.

## Changelog
:cl:
tweak: tweaked name of torp casing
/:cl:

### CODE REVIEW

**Comment by crossedfall** [comment]
Torpedo sizes are still done in mm. E.g the Mark 48 is a 530 mm torpedo.

File: sephora/game/torpedo_components.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "torpedo casing"
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "case"
-	desc = "The outer casing of a 30mm torpedo."
+	desc = "The outer casing of a 70cm torpedo."
```
Comment (crossedfall) [inline sephora/game/torpedo_components.dm]:
```suggestion
	desc = "The outer casing of a 750mm torpedo."
```
Added a little extra for flair and so it wouldn't be perfectly round.

**Comment by Karmic-Skink** [comment]
mm please, what is cm?

**Comment by crossedfall** [comment]
> 
> 
> mm please, what is cm?

You're late to the party, he already changed it.

**Comment by Karmic-Skink** [comment]
> mm please, what is cm?
> 
> You're late to the party, he already changed it.

I'm always late to every party.

**Comment by Vasily2013** [comment]
> mm please, what is cm?

cm is centimeters. Don't you Europeans know your own system of measurement?

**Comment by Kmc2000** [comment]
30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang30 mm gang 30 mm gang 30 mm gang 30 mm gang 30 mm gang30 mm gang 30 mm gang30 mm gang 30 mm gang

**Pull request closed without merging**